package.json

```diff
@@ -37,11 +37,11 @@
     "pify": "^2.2.0",
     "rimraf": "^2.4.3",
     "run-applescript": "^2.0.0",
-    "xdg-trashdir": "^2.0.0"
+    "xdg-trashdir": "^2.1.0",
+    "path-exists": "^2.0.0"
   },
   "devDependencies": {
     "ava": "*",
-    "path-exists": "^2.0.0",
```

Comment:
following this logic, you'd need to list the same dependency 5 times if you use it in 5 places ;-)

If you remove a `dependency` that you use in prod. and you still need it in `dev` - you can then move it back to `devDependencies`. You should not list the same dep. more than once.